src/robot/htmldata/rebot/log.js

```diff
@@ -62,6 +62,16 @@ function expandElementWithId(elementid) {
     expandElement(window.testdata.findLoaded(elementid));
 }
 
+function expandElementsWithIds(ids) {
+    util.map(ids,expandElementWithId)
```

Comment:
Space needed after comma and semicolon at the end. Semicolon missing also from the next function. #nitpick